config/default.yml

```
@@ -3422,6 +3422,11 @@ Style/OneLineConditional:
   VersionAdded: '0.9'
   VersionChanged: '0.38'
 
+Style/OpenSSLDigestConstant:
+  Description: "Don't use algorithm constants for OpenSSL::Digest."
+  Enabled: true
```

Can you specify `Enabled: pending` instead of `Enabled: true`?

```suggestion
  Enabled: pending
```